website/source/docs/job-specification/vault.html.md

```diff
@@ -50,13 +50,8 @@ The Nomad client will make the Vault token available to the task by writing it
 to the secret directory at `secret/vault_token` and by injecting a VAULT_TOKEN
 environment variable.
 
-If Vault token renewal fails due to a Vault outage, the Nomad client will
```

Comment:
was all of this a typo? seems unlikely

Reply:
@jippi - first commit was the typo, found that duplicated text after.